apps/emqx_ft/test/emqx_ft_api_SUITE.erl

```diff
@@ -310,17 +275,161 @@ t_ft_disabled(Config) ->
+            Config
+        )
+    ),
+    S3Exporter = #{
+        <<"host">> => <<"localhost">>,
+        <<"port">> => 9000,
+        <<"bucket">> => <<"emqx">>,
+        <<"transport_options">> => #{
+            <<"ssl">> => #{
+                <<"enable">> => true,
+                <<"certfile">> => emqx_ft_test_helpers:pem_privkey(),
+                <<"keyfile">> => emqx_ft_test_helpers:pem_privkey()
+            }
+        }
+    },
+    ?assertMatch(
+        {ok, 200, #{
+            <<"enable">> := true,
+            <<"storage">> := #{
+                <<"local">> := #{
+                    <<"exporter">> := #{
+                        <<"s3">> := #{
+                            <<"transport_options">> := #{
+                                <<"ssl">> := #{
+                                    <<"enable">> := true,
+                                    <<"certfile">> := <<"/", _CertFilepath/bytes>>,
+                                    <<"keyfile">> := <<"/", _KeyFilepath/bytes>>
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }},
+        request_json(
+            put,
+            uri(["file_transfer"]),
+            #{
+                <<"enable">> => true,
+                <<"storage">> => #{
+                    <<"local">> => #{
+                        <<"exporter">> => #{
+                            <<"s3">> => S3Exporter
+                        }
+                    }
+                }
+            },
+            Config
+        )
+    ),
+    ?assertMatch(
+        {ok, 400, _},
+        request_json(
+            put,
+            uri(["file_transfer"]),
+            #{
+                <<"enable">> => true,
+                <<"storage">> => #{
+                    <<"local">> => #{
+                        <<"exporter">> => #{
+                            <<"s3">> => emqx_utils_maps:deep_put(
+                                [<<"transport_options">>, <<"ssl">>, <<"keyfile">>],
+                                S3Exporter,
+                                <<>>
+                            )
+                        }
+                    }
+                }
+            },
+            Config
+        )
+    ),
+    ?assertMatch(
+        {ok, 200, #{}},
+        request_json(
+            put,
+            uri(["file_transfer"]),
+            #{
+                <<"enable">> => true,
+                <<"storage">> => #{
+                    <<"local">> => #{
+                        <<"exporter">> => #{
+                            <<"s3">> => emqx_utils_maps:deep_put(
+                                [<<"transport_options">>, <<"ssl">>, <<"enable">>],
+                                S3Exporter,
+                                false
+                            )
+                        }
+                    }
+                }
+            },
+            Config
+        )
+    ),
+    ok.
```

Comment:
Q: maybe we could add a couple cases to test the API `filter` (e.g.: `GET`ting the file list when the feature is disabled)?

Reply:
There are some: https://github.com/emqx/emqx/pull/10986/files/15ce324c1d3df85b5f92f060a6ef7d6e389aa439#diff-d6b72b39135941b2198bd57c769e916e6ff8e0debb49c3430e9841c7fb889609R247

Reply:
`t_ft_disabled` case

Reply:
Cool, I thought that change was just introduced along with the swagger changes.